tests/unit/test_configuration.py

```diff
@@ -1,51 +1,120 @@
+    config = get_configuration()
+    # Check forms
+    assert set(f.name for f in FORMS) == set(config.keys())
+    for form in FORMS:
+        # Check row noise
+        for row_noise in form.row_noise_types:
+            config_probability = config[form.name].row_noise[row_noise.name].probability
+            default_probability = (
+                DEFAULT_NOISE_VALUES.get(form.name, {})
+                .get("row_noise", {})
+                .get(row_noise.name, {})
+                .get("probability", "no default")
+            )
+            if default_probability == "no default":
+                assert config_probability == getattr(NOISE_TYPES, row_noise.name).probability
+            else:
+                assert config_probability == default_probability
+        for col in form.columns:
+            for noise_type in col.noise_types:
+                config_level = config[form.name].column_noise[col.name][noise_type.name]
+                baseline_level = getattr(NOISE_TYPES, noise_type.name)
+                # FIXME: Is there a way to allow for adding new keys when they
+                # don't exist in baseline? eg the for if loops below depend on their
+                # being row_noise, token_noise, and additional parameters at the
+                # baseline level ('noise_type in col.noise_types')
+                # Would we ever want to allow for adding non-baseline default noise?
+                if noise_type.row_noise_level:
+                    config_row_noise_level = config_level.row_noise_level
+                    default_row_noise_level = (
+                        DEFAULT_NOISE_VALUES.get(form.name, {})
+                        .get("column_noise", {})
+                        .get(col.name, {})
+                        .get(noise_type.name, {})
+                        .get("row_noise_level", "no default")
+                    )
+                    if default_row_noise_level == "no default":
+                        assert config_row_noise_level == baseline_level.row_noise_level
+                    else:
+                        assert config_row_noise_level == default_row_noise_level
+                if noise_type.token_noise_level:
+                    config_token_noise_level = config_level.token_noise_level
+                    default_token_noise_level = (
+                        DEFAULT_NOISE_VALUES.get(form.name, {})
+                        .get("column_noise", {})
+                        .get(col.name, {})
+                        .get(noise_type.name, {})
+                        .get("token_noise_level", "no default")
+                    )
+                    if default_token_noise_level == "no default":
+                        assert config_token_noise_level == baseline_level.token_noise_level
+                    else:
+                        assert config_token_noise_level == default_token_noise_level
+                if noise_type.additional_parameters:
```

Comment:
I re-worked this to be more inline with how we are doing the previous checks. It also looks at the DEFAULT_NOISE_VALUES now which it previously didn't.